src/hotspot/share/gc/shenandoah/shenandoahTaskqueue.hpp

```diff
@@ -169,8 +159,12 @@ class ObjArrayChunkedTask
     return (int) ((val >> pow_shift) & right_n_bits(pow_bits));
   }
 
+  inline bool decode_not_chunked(uintptr_t val) const {
+    return (val & ~right_n_bits(oop_bits + pow_bits)) == 0;
+  }
+
   inline uintptr_t encode_oop(oop obj) const {
-    return ((uintptr_t)(void*) obj) << oop_shift;
+    return ((uintptr_t) (void *) obj) << oop_shift;
```

Comment:
Looks like an extra space sneaked in here. Or was that intentional?

Reply:
Not intentional, fixed.